requirements.txt

```diff
@@ -1 +1 @@
-ops >= 1.2.0
+git+https://github.com/canonical/operator/#egg=ops
```

Comment:
I'm not sure if it's enough for this charm requirements, but we can change to `ops >= 1.3.0` instead of `git`'s master version.

Reply:
I've updated to v1.3.0, manually tested it, and run integration and unit tests, which all pass.